MC38472 remote storage (#8254)
magento/devdocs: src/guides/v2.4/install-gde/install/cli/install-cli-install.md

magento setup:install --<option>=<value> ... --<option>=<value>
```

> The following tables describe the installation option names and values. For example installation commands. See [Sample localhost installations](#install-cli-example).

Reviewer: ```suggestion
The following tables describe the installation option names and values, such as installation commands. See [Sample localhost installations](#install-cli-example).
```
